src/presign/record.rs

```diff
@@ -65,7 +63,10 @@ impl TryFrom<RecordPair> for PresignRecord {
             return Err(InternalInvariantFailed);
         }
 
-        let delta_inv = Option::<Scalar>::from(delta.invert()).ok_or(CouldNotInvertScalar)?;
+        let delta_inv = Option::<Scalar>::from(delta.invert()).ok_or_else(|| {
+            error!("Could not invert delta as it is 0. Either you got profoundly unlucky or more likely there's a bug");
+            InternalInvariantFailed
+        })?;
```

Comment:
question: some of those deltas that are summed up to make this are from other parties, right? I might say `ProtocolError` here.